manifests/user.pp

```diff
@@ -42,23 +43,27 @@
 #   able to use password authentication on splatnix (*nix: Linux, BSD, macOS,
 #   and Solaris).
 #
-#   Windows requires passwords. If it is not specified, this module will remove
-#   the user account.
+#   You may specify this in hiera under `account::user` parameter. See the
+#   [Passwords section in README.md](https://github.com/ploperations/ploperations-account/blob/master/README.md#passwords).
+#
+#   Windows requires passwords. If it is not specified here or in hiera, this
+#   will remove the user account.
 # @param shared_accounts
-#   An array of shared accounts to add the user's SSH key to.
+#   An array of shared accounts to add the user's SSH key to. To activate,
+#   collect the `Ssh::Authorized_key` virtual resources in a profile, e.g:
+#
+#   ~~~
+#   Ssh::Authorized_key <| tag == "${shared_account}-keys" |>
+#   ~~~
+#
+#   See the
+#   [Shared accounts section in README.md](https://github.com/ploperations/ploperations-account/blob/master/README.md#shared-accounts).
 define account::user (
   Enum['present', 'absent']  $ensure             = 'present',
   Optional[String[1]]        $group              = undef,
   Array[String[1]]           $groups             = [],
   Optional[String]           $comment            = undef,
-  Enum[
-    '/bin/bash',
-    '/bin/sh',
-    '/bin/zsh',
-    '/bin/false',
-    '/usr/sbin/nologin',
-    '/usr/bin/git-shell'
-  ]                          $shell              = '/bin/bash',
+  Stdlib::Unixpath           $shell              = '/bin/bash',
```

Comment:
The docs still list individual shells... does that need correcting?

Reply:
I don't see them listed anywhere in the docs, but maybe I inadvertently fixed it?

Reply:
Looks like a run of `puppet strings` probably got it :)